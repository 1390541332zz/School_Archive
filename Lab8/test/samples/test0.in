load test/samples/test0.data
location 0 0
location 12 1
location -1 -1
location 13 2