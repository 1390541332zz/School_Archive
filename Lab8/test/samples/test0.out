 0,  0: 2
12,  1: \
-1, -1: bad location
13,  2: bad location